applications/ChimeraApplication/python_scripts/chimera_modelpart_import.py

```diff
@@ -0,0 +1,286 @@
+import KratosMultiphysics
+import KratosMultiphysics.ChimeraApplication as KratosChimera
+import numpy as np
+import os
+import json
+from copy import deepcopy
+
+def ImportChimeraModelparts(main_modelpart, chimera_mp_import_settings_list, material_file="", parallel_type="OpenMP"):
```

Comment:
Shouldn't this be implemented in the `ImportModelPart` of the Chimera solver?

Reply:
Ya well instead of putting all this implementation into `ImportModelPart`  I made a separate module and using this module in `ImportModelPart` !

Reply:
OK then.